Documentation/Books/Manual/Programs/Arangorestore/ParallelRestore.md

```diff
@@ -176,15 +176,15 @@ coordinator_0.sh, coordinator_1.sh, coordinator_2.sh).
 
 The `coordinator_<number-of-coordinator>.sh` scripts, that were created in the
 previous script, now have to be executed on each machine where a coordinator
-is running. This will start a parallel restore of the `dump`.
+is running. This will start a parallel restore of the dump.
 
-### Reverse to the initial Replication Factor
+### Revert to the initial Replication Factor
 
 Once the _arangorestore_ process on every coordinator is completed, the
 replication factor has to be set to its initial value.
 
 Run the following command from exactly one coordinator (any coordinator can be
-used) - Please adjust the `replicationFactor` value accordingly.:
+used) - please adjust the `replicationFactor` value accordingly.:
```

Comment:
There's still an extra . before the colon, which doesn't seem to serve any purpose

Reply:
done